src/Contract/Scripts.purs

```diff
@@ -124,9 +123,6 @@ applyArgsM a = map hush <<< applyArgs a
 mintingPolicyHash :: MintingPolicy -> Contract (Maybe MintingPolicyHash)
 mintingPolicyHash = wrap <<< Scripts.mintingPolicyHash
 
-scriptHash :: PlutusScript -> Contract (Maybe Hash.ScriptHash)
```

Comment:
Given the new set up, could you add `scriptHash` to ` module ExportScripts` and remove from `module Scripts`?